src/main/java/de/tum/in/www1/artemis/web/rest/ExerciseGroupResource.java

```diff
@@ -0,0 +1,182 @@
+        log.debug("REST request to get exercise group : {}", exerciseGroupId);
+        Optional<ResponseEntity<ExerciseGroup>> accessFailure = examAccessService.checkCourseAndExamAndExerciseGroupAccess(courseId, examId, exerciseGroupId);
+        return accessFailure.orElseGet(() -> ResponseEntity.ok(exerciseGroupService.findOne(exerciseGroupId)));
+    }
+
+    /**
+     * GET courses/{courseId}/exams/{examId}/exerciseGroups : Get all exercise groups of the given exam
+     *
+     * @param courseId  the course to which the exercise groups belong to
+     * @param examId    the exam to which the exercise groups belong to
+     * @return the ResponseEntity with status 200 (OK) and a list of exercise groups. The list can be empty
+     */
+    @GetMapping("courses/{courseId}/exams/{examId}/exerciseGroups")
+    @PreAuthorize("hasAnyRole('ADMIN', 'INSTRUCTOR')")
+    public ResponseEntity<List<ExerciseGroup>> getExerciseGroupsForExam(@PathVariable Long courseId, @PathVariable Long examId) {
+        log.debug("REST request to get all exercise groups for exam : {}", examId);
+        Optional<ResponseEntity<List<ExerciseGroup>>> courseAndExamAccessFailure = examAccessService.checkCourseAndExamAccess(courseId, examId);
+        return courseAndExamAccessFailure.orElseGet(() -> ResponseEntity.ok(exerciseGroupService.findAllByExamId(examId)));
+    }
+
+    /**
+     * DELETE /courses/{courseId}/exams/{examId}/exerciseGroups/{exerciseGroupId} : Delete the exercise group with the given id.
+     *
+     * @param courseId          the course to which the exercise group belongs to
+     * @param examId            the exam to which the exercise group belongs to
+     * @param exerciseGroupId   the id of the exercise group to delete
+     * @return the ResponseEntity with status 200 (OK)
+     */
+    @DeleteMapping("/courses/{courseId}/exams/{examId}/exerciseGroups/{exerciseGroupId}")
+    @PreAuthorize("hasAnyRole('ADMIN', 'INSTRUCTOR')")
+    public ResponseEntity<Void> deleteExerciseGroup(@PathVariable Long courseId, @PathVariable Long examId, @PathVariable Long exerciseGroupId) {
+        log.info("REST request to delete exercise group : {}", exerciseGroupId);
+
+        Optional<ResponseEntity<Void>> accessFailure = examAccessService.checkCourseAndExamAndExerciseGroupAccess(courseId, examId, exerciseGroupId);
+        if (accessFailure.isPresent()) {
+            return accessFailure.get();
+        }
+
+        ExerciseGroup exerciseGroup = exerciseGroupService.findOne(exerciseGroupId);
+
+        User user = userService.getUser();
+        AuditEvent auditEvent = new AuditEvent(user.getLogin(), Constants.DELETE_EXERCISE_GROUP, "exerciseGroup=" + exerciseGroup.getTitle());
+        auditEventRepository.add(auditEvent);
+        log.info("User " + user.getLogin() + " has requested to delete the exercise group {}", exerciseGroup.getTitle());
+
+        exerciseGroupService.delete(exerciseGroupId);
```

Comment:
I don't think this will work, if there already are exercises connected with the exercise group.
You would first need to delete the exercises properly. Looking into deleteCourse, the following code should work.

```
        for (Exercise exercise : exerciseGroup.getExercises()) {
            exerciseService.delete(exercise.getId(), false, false);
        }
```

This would mean that you need to load the exerciseGroup above "withExercises()"

I suggest we change this in a follow-up PR and add a server integration test case to cover this case!